index.js

```diff
@@ -38,6 +40,13 @@ module.exports = Harvest = function (opts) {
         baseURL: self.host
     }, {
         run: function (type, url, data) {
+            if(self.use_oauth) {
+                if(!self.access_token) {
+                    throw new Error("An access token is required if using oAuth, use parseAccessCode or pass an access_token before making any requests");
+                }
+                url = url + "?access_token=" + self.access_token;
```

Comment:
Hi,

I was very interested in using this tool for an application that used oAuth and was very happy to see you had a pull request for it! I've added your changes into my current system, however I saw that this line caused some errors when a URL already had query parameters. Here's my suggestion for this line to be used instead:
```javascript
url = url.indexOf('?') > -1 ? url + "&access_token=" + self.access_token : url + "?access_token=" + self.access_token;
```